apps/src/pythonlab/types.ts

```diff
@@ -24,3 +25,9 @@ export type MessageType =
   | 'system_error'
   | 'loading_pyodide'
   | 'loaded_pyodide';
+
+export interface PythonlabLevelProperties extends LevelProperties {
+  // An optional level property that determines whether to display the
+  // 'Send to micro:bit' header button on pythonlab levels.
+  enableMicroBit?: boolean;
```

Comment:
I think we should move this property to be an optional field on `LevelProperties` rather than creating a separate type we have to cast to. We have other fields already on `LevelProperties` that are not relevant to all labs.